tests/ui/EditTaskHelpers.test.ts

```diff
@@ -49,8 +49,10 @@ describe('labelContentWithAccessKey() tests', () => {
 });
 
 describe('parseAndValidateRecurrence() tests', () => {
-    const emptyTask = new TaskBuilder().build();
-    const { editableTask } = EditableTask.fromTask(emptyTask, [emptyTask]);
+    const taskWithoutRecurrence = new TaskBuilder().build();
+    const { editableTask: editableTaskWithoutRecurrence } = EditableTask.fromTask(taskWithoutRecurrence, [
+        taskWithoutRecurrence,
+    ]);
```

Comment:
I'm really sorry, but I understood the previous code very clearly.

Whereas the new code, I have read at least half a dozen times, and have no idea at a glance what it is doing.
Is it declaring a function?
'without recurrence' is mentioned 4 times on 3 lines - but what is the actual behaviour?

I don't feel that the code review was actually beneficial in this case, after all!

A simpler change would be to go back the previous version, and change line 52 to make it honest:

```ts
const emptyTask = new TaskBuilder().description('').build();
```